conda/core/index.py

```diff
@@ -38,41 +36,22 @@
 
 log = getLogger(__name__)
 
+LAST_CHANNEL_URLS = []
+
 
+@deprecated(
+    "25.1",
+    "25.7",
```

Comment:
The `conda` deprecation formula:

`(the next release + 4 months) round up to nearest .3 or .9`

25.1 + 4 months = 25.5
25.5 round up to nearest .3 or .9 = 25.9

```suggestion
    "25.9",
    "26.3",
```